pages/1.9/release-notes/1.9.8/index.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.9.8
+title: Release Notes for 1.9.8
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.9.8.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.8/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.9.8
+
+- COPS-1879 - Fixed the failure to kill a docker task in staging.
```

Comment:
Minor tweak: "Fixed a failure to kill a Docker task in staging."

Reply:
Changes fixed/added for this comment.